subsys/bluetooth/controller/Kconfig.df

```diff
@@ -0,0 +1,119 @@
+# Zephyr Bluetooth Controller configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DF_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_TX_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_RX_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_RX_SAMPLING_1US_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_2US_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_1US_SUPPORT
+	bool
+
+menuconfig BT_CTLR_DF
+	bool "LE Direction Finding [Experimental]"
+	depends on BT_CTLR_DF_SUPPORT
+	help
+	  Enable support for Bluetooth 5.1 Direction Finding
+
+if BT_CTLR_DF
+
+# Basic controller functionalities required for implementation of
+# Bluetooth v5.1 Direction Finding
+
+config BT_CTLR_DF_CTE_TX
+	bool "Enable transmission of Constant Tone Extension"
+	depends on BT_CTLR_DF_CTE_TX_SUPPORT
+	default y
+	help
+	  Enable support for transmission of Constant Tone Extension in
+	  controller.
+
+config BT_CTLR_DF_CTE_RX_SAMPLING_1US
+	bool "Enable reception of CTE with 1us sampling slots"
+	depends on BT_CTLR_DF_CTE_RX_SAMPLING_1US_SUPPORT
+	default y
+	help
+	  Enable reception of Constant Tone Extension. Process IQ sampling
+	  during CTE reception with 1us sampling slots.
+	  This sampling type is optional for Direction Finding according to
+	  Bluetooth v5.1.
+
+config BT_CTLR_DF_ANT_SWITCHING_1US
+	bool "Enable support for 1us antenna switching slots"
+	depends on BT_CTLR_DF_ANT_SWITCHING_1US_SUPPORT
+	default y
+	help
+	  Enable support for 1us antenna switching slots. This antenna switching
+	  mode is optional for Direction Finding according to Bluetooth v5.1.
+
+# Features related with Direction Finding
+# BT Core spec 5.1, Vol 6, Part B, sec. 4.6 Feature Support
+
+config BT_CTLR_DF_ANT_SWITCHING_TX
+	bool "Enable antenna switching during CTE transmission (AoD) feature"
+	depends on BT_CTLR_DF_CTE_TX && BT_CTLR_DF_ANT_SWITCHING_2US_SUPPORT
+	default y
+	help
+	  Enable support for antenna switching during CTE transmission.
+	  Also known as Angle of Departure mode.
+
+config BT_CTLR_DF_ANT_SWITCHING_RX
+	bool "Enable antenna switching during CTE reception (AoA) feature"
+	depends on BT_CTLR_DF_ANT_SWITCHING_2US_SUPPORT
+	default y
+	help
+	  Enable support for antenna switching during CTE reception.
+	  Also known as Angle of Arrival mode.
+
+config BT_CTLR_DF_CTE_RX
+	bool "Enable receive of Constant Tone Extension feature"
+	depends on BT_CTLR_DF_CTE_RX_SUPPORT
+	default y
+	help
+	  Enable support for reception of Constant Tone Extension in controller.
+
+config BT_CTLR_DF_CONN_CTE_REQ
+	bool "Enable Connection CTE Request feature"
+	depends on BT_CTLR_DF_CTE_RX && BT_CONN
+	help
```

Comment:
RSP is default y, should this also be that?

Reply:
Added missing default